### PR TITLE
Added create inventory item, fixed database bugs and tested get inventory item

### PR DESCRIPTION
**Pull request by vincezh2000**
Made the following changes:
- added create endpoint in "routes.py"
- added more type checks in "models.py"
- fixed type mismatch problem in "factory.py"

### CODE REVIEW

File: tests/test_routes.py [hunk 1]
```diff
@@ -82,39 +85,33 @@ def test_index(self):
...
     def test_create_inventory_item(self):
-        """It should Create a new Inventory Item"""
-        item = InventoryItemFactory()
-        logging.debug(item)
-        resp = self.client.post(
-            "/inventory", json=item.serialize(), content_type="application/json"
-        )
-        self.assertEqual(resp.status_code, status.HTTP_201_CREATED)
-        location = resp.headers.get("Location")
+        """It should Create a new Pet"""
```
Comment (akash-belide) [inline tests/test_routes.py]: It says "Pet" you have to change it.

Comment (vincezh2000) [reply tests/test_routes.py]: Fixed

File: .idea/.gitignore
Comment (Ravi-Chavda) [inline .idea/.gitignore:1]: File not required

File: .idea/inspectionProfiles/Project_Default.xml
Comment (Ravi-Chavda) [inline .idea/inspectionProfiles/Project_Default.xml:1]: local file

**Review by Ravi-Chavda — APPROVED** [review]

**Pull request merged**